### PR TITLE
Don't indent embedded scripts/styles

### PR DESCRIPTION
**Pull request by azz**

### CODE REVIEW

File: src/multiparser.js [hunk 1]
```diff
@@ -163,6 +165,10 @@ function getText(options, node) {
   return options.originalText.slice(util.locStart(node), util.locEnd(node));
 }
 
+function deIndent(options, doc) {
+  return align(-options.tabWidth, doc);
```
Comment (azz) [inline src/multiparser.js]: Is it ok to use a negative number for alignment?

Comment (vjeux) [inline src/multiparser.js]: I've been trying to avoid doing that. Could you instead not indent when you are about to print them like you do for html?

Comment (azz) [inline src/multiparser.js]: The indentation comes from the parent node in the AST, which currently has no knowledge that a child might switch to a different parser.

**Comment by hawkrives** [comment]
I like indenting the contents of my script/style tags…

Can we discuss this before merging it? 😅 Thanks

**Comment by hawkrives** [comment]
Now I see where this PR came from: https://github.com/prettier/prettier/pull/2093#discussion_r121271048

@vjeux, you never indent your `<script>` or `<style>` tags? I like to indent them… I think because it feels similar to indenting any other tag's contents.

**Comment by azz** [comment]
@hawkrives Sure! I personally don't mind either way, I was just modelling this:

image

**Comment by azz** [comment]
If it requires doing somewhat hacky things maybe it's better to leave it for now and see how people feel?

**Comment by vjeux** [comment]
sgtm

**Comment by hawkrives** [comment]
I don't actually know how to debate this – I think it's probably completely subjective 😅.

I think my objection comes down to, it feels inconsistent to only indent tags that contain HTML, but not JS or CSS. IDK.

Thanks for humoring me 😸

**Pull request closed without merging**